OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -811,7 +811,15 @@ void IResolveOrder.ResolveOrder(Actor self, Order order)
 					self.CancelActivity();
 
 				self.SetTargetLine(Target.FromCell(self.World, cell), Color.Green);
-				self.QueueActivity(order.Queued, WrapMove(new Move(self, cell, WDist.FromCells(8), null, true)));
+
+				var selectionCount = self.World.Selection.ActorCount;
+
+				// Scale nearEnough by the formula square root of world selection divided by 2 + half a cell safety margin.
```

Comment:
I think this should divide by Sqrt(2) instead of 2. That gives you the half-diagonal instead of the half-side of the minimal square formation.